core/services/validators.go

```diff
@@ -133,10 +132,37 @@ func validateFluxMonitor(i models.Initiator, j models.JobSpec) error {
 	} else if i.PollingInterval < MinimumPollingInterval {
 		fe.Add("pollingInterval must be equal or greater than " + MinimumPollingInterval.String())
 	}
+	if err := validateFeeds(i.Feeds, store); err != nil {
+		fe.Add(err.Error())
+	}
 
 	return fe.CoerceEmptyToNil()
 }
 
+func validateFeeds(feeds models.Feeds, store *store.Store) error {
+	var feedsData []interface{}
+	if err := json.Unmarshal(feeds.Bytes(), &feedsData); err != nil {
+		return errors.New("invalid json for feeds parameter")
+	}
+	if len(feedsData) == 0 {
+		return errors.New("no feeds")
+	}
+
+	var bridgeNames []string
+	for _, entry := range feedsData {
+		switch entry.(type) {
```

Comment:
Should this validate URL's too? And have a `default` clause, for unrecognized types?